src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -957,11 +955,350 @@ private class ACCT_IndividualAccounts_TEST {
 
         List<contact> updatedContacts = [SELECT Account.Name, AccountId, Account.RecordTypeId FROM Contact where id=:con.id];
 
-        //The account should not be the one we started with        
+        //The account should not be the one we started with
         System.assertNotEquals(NULL, updatedContacts[0].AccountId);
         System.assertNotEquals(acct.id, updatedContacts[0].AccountId);
         System.assertEquals(UTIL_Describe_API.getAdminAccRecTypeID(), updatedContacts[0].Account.RecordTypeId);
+    }
+
+    /*********************************************************************************************************
+    * @description Test Method for when a Contact's owner change the Account's Owner
+    * should match the Contact's new owner.
+    */
+    @isTest
+    private static void contactOwnerChange() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                      (Account_Processor__c = UTIL_Describe_API.getAdminAccRecTypeID()));
+
+		Id sysAdminProfileId = UTIL_Profile.getInstance().getProfileIds(UTIL_Profile.SYSTEM_ADMINISTRATOR)[0];
+        User adminUser = new User(LastName = 'TestAdmin', Email = 'testAdminUser@emailtest.edatest',
+									ProfileId = sysAdminProfileId, isActive = true, UserName = 'testAdminUser@emailtest.edatest',
+         							Alias = 'tAdmin0', TimeZoneSidKey = 'America/Los_Angeles',
+                					LocaleSidKey = 'en_US', LanguageLocaleKey = 'en_US', EmailEncodingKey = 'ISO-8859-1');
+        insert adminUser;
 
+        Contact con = UTIL_UnitTestData_API.getContact();
+        insert con;
+
+		con.OwnerId = adminUser.Id;
```

Comment:
This was resolved, but I still see issues here... it's just Github though. I'm looking in VS Code and notice that it's because of using tabs instead of spacing. I might commit up my own quick formatting fixes on this.

Reply:
https://stackoverflow.com/questions/33974681/how-can-i-convert-tabs-to-spaces-and-vice-versa-in-an-existing-file